services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:
```

Comment:
First of all it not return one object
Also it does not return objects, it return QuerySet